modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -151,7 +152,7 @@ public function setUp()
                 'ID'                    => '111111',
                 'CandID'                => '999888',
                 'CenterID'              => '55',
-                'RegistrationProjectID' => '7777',
+                'ProjectID'             => '7777',
```

Comment:
Isn't `RegistrationProjectID` the newer change?

Reply:
this is in the session table